Explorer/Assets/DCL/SDKComponents/SceneUI/Defaults/PBUiTransform_Defaults.cs

```diff
@@ -19,5 +21,17 @@ public static YGAlign GetAlignContent(this PBUiTransform self) =>
 
         public static YGWrap GetFlexWrap(this PBUiTransform self) =>
             self.HasFlexWrap ? self.FlexWrap : YGWrap.YgwNoWrap;
+
+        public static Color GetBorderBottomColor(this PBUiTransform self) =>
+            self.BorderBottomColor?.ToUnityColor() ?? ColorDefaults.COLOR_TRANSPARENT;
+
+        public static Color GetBorderTopColor(this PBUiTransform self) =>
+            self.BorderBottomColor?.ToUnityColor() ?? ColorDefaults.COLOR_TRANSPARENT;
+
+        public static Color GetBorderLeftColor(this PBUiTransform self) =>
+            self.BorderBottomColor?.ToUnityColor() ?? ColorDefaults.COLOR_TRANSPARENT;
+
+        public static Color GetBorderRightColor(this PBUiTransform self) =>
+            self.BorderBottomColor?.ToUnityColor() ?? ColorDefaults.COLOR_TRANSPARENT;
```

Comment:
is it a small copy-paste mistake that you didn't change `self.BorderBottomColor` in these methods? If not, then merge it into one please (or correct)

Reply:
Thanks! addressed at https://github.com/decentraland/unity-explorer/pull/3646/commits/beb4b1c6008b120d52f9b0f50e60b9e1f3f29b17